runtime-common/src/pallets.rs

```diff
@@ -0,0 +1,23 @@
+/// Name of the With-Darwinia messages pallet instance that is deployed at bridged chains.
```

Comment:
nitpick: Add license. Please also check other newly added files.

Reply:
Another concern: Where are those pallet names const are used? It's very easy to forget to sync the changes if rename occurs on the runtime side.

Reply:
The same as https://github.com/paritytech/parity-bridges-common/blob/f924be9867bf381c76f1860b60aac4d6d4c58b24/primitives/chain-kusama/src/lib.rs#L35.

And we won't change a pallet name frequently.

Anyway, we need a place to storeh this list.

Reply:
Do not forget to add a license header.